common/v2/features/DeFiZap/config.ts

```diff
@@ -131,3 +143,78 @@ export const ZAPS_CONFIG: IZapConfigObject = {
     positionDetails: () => BearishIndicator({ text: 'Bearish on ETH' })
   }
 };
+
+export const riskAndReward: RiskAndReward[] = [
```

Comment:
I want to have some other people go through this copy at some point, because it sounds odd atm.

Reply:
I could just do it in JSX, was just repetitive.